.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '42 5 * * *'
+
+jobs:
+  testsuite:
+    name: all tests
+    runs-on: ubuntu-20.04
+    strategy:
+      # This prevents cancellation of matrix job runs, if one or more already failed
+      # and let the remaining matrix jobs be be executed anyway.
+      fail-fast: false
+      matrix:
+        # @todo: Change back to to php: [ '8.1', '8.2' ] if JWT token library from core is working PHP8.2
```

Comment:
The least supported PHP version (for v12 PHP 8.1, or whatever we agree upon) should be sufficient. No need to run with different PHP versions.

For example, if we would use readonly classes (introduced in PHP 8.2, and would be fine in my opinion), this should be set to PHP 8.2. PHP 8.1 would then run in an error.